### PR TITLE
Molo 317: Implement Mote

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

File: molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/mote/projects/{{cookiecutter.app_name}}/website/src/patterns/basics/copyright/element.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{% raw %}
+{% load mote_tags %}
+
+<div class="footer">
+    <p class="footer__copyright">
+      © 2016 UNFPA, Ford Foundation. All Rights Reserved.
```
Comment (saeedmb8) [inline molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/mote/projects/{{cookiecutter.app_name}}/website/src/patterns/basics/copyright/element.html]: Can we use a generic text here, maybe just `© All Rights Reserved.` or something like that

File: molo/core/styles/modules/_modules.scss [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+@import "polls/_m-poll-right";
+@import "polls/_m-poll";
+
+@import "profiles/_m-profile-form";
+@import "profiles/_m-profile-view";
+@import "profiles/_m-profile";
+
+@import "yourwords/_m-yourwords-article";
+@import "yourwords/_m-yourwords-secondary";
+@import "yourwords/_m-yourwords";
```
Comment (saeedmb8) [inline molo/core/styles/modules/_modules.scss]: can we remove these plugins import so we won't forget to remove then and we also won't add anything related to the plugins in molo.core

File: molo/core/styles/modules/comments/_m-comment-profile.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
+.comment-list__profile,
```
Comment (saeedmb8) [inline molo/core/styles/modules/comments/_m-comment-profile.scss]: same here the CSS for the commenting plugin

File: molo/core/templates/core/article_page.html [hunk 4]
```diff
@@ -1,47 +1,16 @@
...
-      {% endif %}
-    {% endfor %}
-</article>
-{% if settings.core.SiteSettings.enable_clickable_tags %}
-    {% for tag in self.tags_list %}
-        <a href="{% url 'tags_list' tag %}">{{tag}}</a>
-    {% endfor%}
-{% endif %}
+  {% include "patterns/basics/article-content.html" %}
+  {% include "patterns/basics/article-tags.html" %}
```
Comment (saeedmb8) [inline molo/core/templates/core/article_page.html]: can you please add these includes in a `{% block foo %} {% endblock %}` tag

File: molo/core/templates/core/footer_page.html [hunk 5]
```diff
@@ -1,40 +1,35 @@
...
           {{ block }}
       {% endif %}
       {% endif %}
       {% endif %}
-  {% endfor %}
-</article>
+    {% endfor %}
+  </article>
 </div>
+
```
Comment (saeedmb8) [inline molo/core/templates/core/footer_page.html]: should we use something like the`{% include "patterns/basics/article-content.html" %}` here as well?

File: molo/core/templates/core/tags/section_listing_homepage.html [hunk 6]
```diff
@@ -1,25 +1,19 @@
...
+      {% include "patterns/basics/article-list.html" %}
+
+    {% endfor %}
+  {% endif %}
+{% endfor %}
 </div>
-{% endfor%}
 {% endif %}
- 
+<br/><br/>
```
Comment (saeedmb8) [inline molo/core/templates/core/tags/section_listing_homepage.html]: why are we using `<br/>` here?

File: molo/core/templates/patterns/basics/article-meta.html [hunk 7]
```diff
@@ -0,0 +1,16 @@
+{% load wagtailcore_tags core_tags i18n comments molo_commenting_tags }
```
Comment (saeedmb8) [inline molo/core/templates/patterns/basics/article-meta.html]: We don't need this template as well this should be defined in the commenting plugin

File: molo/core/templates/patterns/basics/copyright.html [hunk 8]
```diff
@@ -0,0 +1,7 @@
+{% load i18n %}
+
+<div class="copyright">
+    <p class="footer__copyright">
+      &copy; {% trans "2016 UNFPA, Ford Foundation. All Rights Reserved." %}
```
Comment (saeedmb8) [inline molo/core/templates/patterns/basics/copyright.html]: lets change the copyright text here as well

File: molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/mote/projects/{{cookiecutter.app_name}}/website/src/patterns/basics/article-teaser-meta/element.html [hunk 9]
```diff
@@ -0,0 +1,14 @@
+{% raw %}
+{% load mote_tags %}
+
+<div class="teaser-meta-list">
+  <div class="teaser-meta-list__likes-counter">
+    63
+    <span class="teaser-meta-list__liked-label">likes</span>
+  </div>
```
Comment (saeedmb8) [inline molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/mote/projects/{{cookiecutter.app_name}}/website/src/patterns/basics/article-teaser-meta/element.html]: I don't think we should have `likes` as we are not supporting them in molo

File: molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/mote/projects/{{cookiecutter.app_name}}/website/src/patterns/basics/article-teaser-meta/element.html [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+  <div class="teaser-meta-list__comment-counter">
+    12
+    <span class="teaser-meta-list__comment-label">comments</span>
+  </div>
+</div>
```
Comment (saeedmb8) [inline molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/mote/projects/{{cookiecutter.app_name}}/website/src/patterns/basics/article-teaser-meta/element.html]: lets remove the comment counter from here, we need to add this in the molo.commenting

File: molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/mote/projects/{{cookiecutter.app_name}}/website/src/patterns/basics/form/element.html [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+
+<fieldset>
+  <legend class="checkbox__legend">{{inputs.checkbox_field.label}}</legend>
+  {% for checkboxes in inputs.checkbox_field.choices_list %}
+    <label  class="checkbox__label">
+      <input type="radio" name="choices" value="{{checkboxes.value}}" />
+      {{checkboxes.label}}
+    </label>
+  {% endfor %}
+</fieldset>-->
```
Comment (saeedmb8) [inline molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/mote/projects/{{cookiecutter.app_name}}/website/src/patterns/basics/form/element.html]: can we remove these if we don't need them ?

File: molo/core/styles/modules/articles/_m-article-meta.scss [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+.article-list__reply-counter {
+  @media screen and (min-width: 320px)  {
+    padding: 0 15px 0 10px;
+    float: right;
+  }
+}
+
+.article-list__reply-label {
+  padding: 0 5px 0;
+}
```
Comment (saeedmb8) [inline molo/core/styles/modules/articles/_m-article-meta.scss]: we should remove the css for the likes and comment counter as well

File: molo/core/styles/modules/comments/_m-comment-success.scss [hunk 13]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (saeedmb8) [inline molo/core/styles/modules/comments/_m-comment-success.scss]: here too

File: molo/core/styles/modules/comments/_m-comments.scss [hunk 14]
```diff
@@ -0,0 +1,41 @@
+
```
Comment (saeedmb8) [inline molo/core/styles/modules/comments/_m-comments.scss]: these as well

File: molo/core/styles/modules/menu/_m-menu.scss [hunk 15]
```diff
@@ -0,0 +1,55 @@
...
+    background:  url('../images/icons/search.svg') no-repeat 0 0 / auto 100%;
+    left: 75px;
+  }
+}
+
+#profile_register > .menu-list__anchor {
+  @media screen and (min-width: 320px) {
+    background:  url('../images/icons/user.svg') no-repeat 0 0 / auto 100%;
+  }
+}
```
Comment (saeedmb8) [inline molo/core/styles/modules/menu/_m-menu.scss]: profile related css as well

File: molo/core/styles/modules/polls/_m-poll-image.scss [hunk 16]
```diff
@@ -0,0 +1,64 @@
...
+    }
+  }
+}
+
+.call-to-action__item--poll-image-summit {
+  @media screen and (min-width: 320px) {
+    background: #FE7A43;
+    font-weight: bold;
+  }
+}
```
Comment (saeedmb8) [inline molo/core/styles/modules/polls/_m-poll-image.scss]: Polls css too

File: molo/core/styles/modules/polls/_m-poll-right.scss [hunk 17]
```diff
@@ -0,0 +1,60 @@
...
+    margin-left: 15px;
+    font-weight: bold;
+    font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
+    @media screen and (min-width: 320px) {
+      font-family: Quicksand,sans-serif;
+      background: #fff;
+      color: #AD016C;
+    }
+  }
+}
```
Comment (saeedmb8) [inline molo/core/styles/modules/polls/_m-poll-right.scss]: here as well

File: .gitignore [hunk 18]
```diff
@@ -36,7 +36,8 @@ nosetests.xml
 /ve/
 logs/*.log
 *.pid
-/testapp/
+/molo-tuneme/
```
Comment (Mitso) [inline .gitignore]: This is now testapp

**Pull request merged**